pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/DeleteAPIHybridClusterIntegrationTest.java

```diff
@@ -0,0 +1,247 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.integration.tests;
+
+import com.alibaba.fastjson.JSON;
+import com.alibaba.fastjson.JSONArray;
+import com.alibaba.fastjson.JSONObject;
+import com.linkedin.pinot.controller.helix.ControllerRequestURLBuilder;
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.testng.annotations.BeforeClass;
+import org.testng.annotations.Test;
+import com.linkedin.pinot.common.data.Schema;
+import com.linkedin.pinot.common.utils.FileUploadUtils;
+import com.linkedin.pinot.common.utils.KafkaStarterUtils;
+import junit.framework.Assert;
+
+
+/**
+ * Hybrid cluster integration test that uploads 8 months of data as offline and 6 months of data as realtime (with a
+ * two month overlap) then deletes segments different ways to test delete APIs.
+ *
+ */
+public class DeleteAPIHybridClusterIntegrationTest extends HybridClusterIntegrationTest {
+  private static final Logger LOGGER = LoggerFactory.getLogger(RealtimeClusterIntegrationTest.class);
+  private static final String TABLE_NAME = "mytable";
+  private static final String TENANT_NAME = "TestTenant";
+  private static int nRows;
+
+  @BeforeClass
+  public void setUp() throws Exception {
+    super.setUp();
+    org.json.JSONObject response = postQuery("select count(*) from 'mytable'");
+    org.json.JSONArray aggregationResultsArray = response.getJSONArray("aggregationResults");
+    org.json.JSONObject firstAggregationResult = aggregationResultsArray.getJSONObject(0);
+    String pinotValue = firstAggregationResult.getString("value");
+    nRows = Integer.parseInt(pinotValue);
+  }
+
+  private int numRowsReturned() throws Exception{
+    org.json.JSONObject response = postQuery("select count(*) from 'mytable'");
```

Comment:
you need to select from tableName__OFFLINE, otherwise things will not work when you remove one realtime segment.

Also, derive the table from super class, do not hardcode table name here